### PR TITLE
Add rollover tooltips to show case counts on histogram bars

### LINKED ISSUE #164: Image gallery: mouseover histograms should show bin counts in a tooltip

### PR DESCRIPTION
**Pull request by ronichoudhury**
Closes #164

### CODE REVIEW

**Comment by brianhelba** [comment]
@msmolens PTAL at this.

File: web_external/js/views/body/ImagesView/HistogramScale.js [hunk 1]
```diff
@@ -149,6 +149,16 @@
             height: barHeight
         };
     };
+    HistogramScale.prototype.getFullRect = function () {
+        var barHeight = this.y(this.yMax);
+      console.log('barHeight', barHeight);
```
Comment (brianhelba) [inline web_external/js/views/body/ImagesView/HistogramScale.js]: Remove this?

Comment (ronichoudhury) [inline web_external/js/views/body/ImagesView/HistogramScale.js]: Oops. Done in e49c579.

**Comment by msmolens** [comment]
Nice! Clever solution to use the invisible bars. One issue I see is that when I scale the histogram, the tooltip above the tallest bar is positioned off the top of the bar, or even off the page:
image

**Comment by ronichoudhury** [comment]
Ooh, nice find @msmolens. Will fix!

**Comment by ronichoudhury** [comment]
@msmolens, check out 496f596. A bit of a hack, but it should be ok. Let me know if not!

**Comment by msmolens** [comment]
@ronichoudhury LGTM once it passes the style tests!

**Pull request merged**